.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: java
+
+jdk: oraclejdk8
+
+before_script:
+  - mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V
+
+script:
+  - mvn test -B
```

Comment:
Once dependencies are installed, it compiles the unit-tests and runs them to check for build failures.